Create referral_and_reseller_policy.md page to describe partner and referral partnership policies

tompinckney opened the pull request:
Adding handbook page to describe Referral and Reseller policies

tompinckney committed:
Create referral_and_reseller_policy.md

Adding handbook page to describe Referral and Reseller policies

tompinckney committed:
squash! Prettier

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,85 @@
+**<span style="text-decoration:underline;">Referral Program Summary</span>**
```
Headings in the handbook must be marked up as actual headings, rather than just visually bolded/underlined/etc. That makes them appear in the table of contents on the right of the page with the right hierarchy, is important for accessibility/ADA and required to show up properly in handbook search.
The page always needs to start with a level 1 heading, which in Markdown starts with a single `#` (the number of hashes matches the level of the heading).

It looks like this page was converted from a Google Docs page. In Google Docs, you achieve this by selecting the heading and using the right heading style:

<img width="278" alt="image" src="https://user-images.githubusercontent.com/10532611/200975757-d7b59d91-74de-48fe-bb57-520ec82582e1.png">

Then make sure to remove any other formatting on the heading:

<img width="224" alt="image" src="https://user-images.githubusercontent.com/10532611/200975828-6663ba49-7808-4108-baa0-ec8447c4c9e4.png">

Then after conversion, this line should simply start with a `#`.

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,85 @@
+**<span style="text-decoration:underline;">Referral Program Summary</span>**
+
+- <span style="text-decoration:underline;">Program Purpose</span>: The Sourcegraph Business Development team wants to enlist the partner community to accelerate Sourcegraph’s growth. The primary impetus of referrals should be that Sourcegraph provides meaningful business benefits to the end customers. Referral payments are important part of certain partners’ business models and Sourcegraph will e ...
```
Underlines should be avoided in handbook pages, since these go on the web, and on the web underlines usually represent links. Instead, use only bold or italic.

After conversion, this will be only wrapped in asteriks, instead of the verbose `<span style="text-decoration:underline;">`.

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,85 @@
+
+- <span style="text-decoration:underline;">Program Purpose</span>: The Sourcegraph Business Development team wants to enlist the partner community to accelerate Sourcegraph’s growth. The primary impetus of referrals should be that Sourcegraph provides meaningful business benefits to the end customers. Referral payments are important part of certain partners’ business models and Sourcegraph will e ...
+- <span style="text-decoration:underline;">Partner Nomination:</span> Potential referral partners should be referred to the Head of Business Development
+- <span style="text-decoration:underline;">Agreement</span>: The partner referral agreement is available [here](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.
```
Links need to have a descriptive name for accessibility. Instead of using the word "here", rephrase the sentence and link the part of the sentence that describes the document, e.g.

> **Agreement**: Use our [partner referral agreement template](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,85 @@
+  - Sourcegraph will make referral payments to partner after receipt of an invoice from the partner and 30 days after receipt of payment from the end customer
+- Sourcegraph reserves the right to update program rules at its discretion
+
+**International Reseller Program Promotion:**
```
This would be a _Heading 2_, presumably.

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,85 @@
+- <span style="text-decoration:underline;">Program Purpose</span>: The Sourcegraph Business Development team wants to enlist the partner community to accelerate Sourcegraph’s growth. The primary impetus of referrals should be that Sourcegraph provides meaningful business benefits to the end customers. Referral payments are important part of certain partners’ business models and Sourcegraph will e ...
+- <span style="text-decoration:underline;">Partner Nomination:</span> Potential referral partners should be referred to the Head of Business Development
+- <span style="text-decoration:underline;">Agreement</span>: The partner referral agreement is available [here](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.
+- <span style="text-decoration:underline;">Program Rules:</span>
```
Are the bullet points following this point intended to be nested under it?

tompinckney committed:
Update referral_and_reseller_policy.md

Updating formatting of the pull request

tompinckney committed:
squash! Prettier

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,94 @@
+## Referral Program Summary
```
This is a heading level 2, but the first heading always _has_ to be a heading level 1 ("Heading 1" in Google docs).

All the other headings then need to be demoted one level down too (3->2, etc)

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,94 @@
+
+### Agreement:
+
+The partner referral agreement is available [here](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.
```
The link needs to have a descriptive accessible name).

Please see this comment: https://github.com/sourcegraph/handbook/pull/5542#discussion_r1018559975

> Links need to have a descriptive name for accessibility. Instead of using the word "here", rephrase the sentence and link the part of the sentence that describes the document, e.g.
> 
> > **Agreement**: Use our [partner referral agreement template](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,94 @@
+   2. Sourcegraph to honor this pricing for any deals registered before the end of that 9 month period for another 90 days (3 months)
+5. Existing customer pricing will increase by no more than 5% per year
+
+Partner Reseller contract is [here](https://docs.google.com/document/d/1y8gF3ai8YdDH-iORi_vVeqET9-g11RZxaXurcNmb0-E/edit)
```
Same with this link

felixfbecker commented on content/departments/sales/tools/referral_and_reseller_policy.md:
```diff
@@ -0,0 +1,94 @@
+
+### Referral and Resale Partner Agreement Inventory:
+
+Please find a list of partner agreements [here](https://docs.google.com/spreadsheets/d/1ClTQfHnYuIGmeBPP0Y5j6PDmwNDQLnDo08RqlZxagWI/edit#gid=0)
```
Same with this link

tompinckney committed:
Update to referral_and_reseller_policy.md

Updating referral_and_reseller_policy.md page to resolve errors before publishing

tompinckney committed:
squash! Prettier

tompinckney committed:
Merge branch 'main' into tompinckney-patch-1
